templates/5_rename_and_reorganize.ipynb

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
<p>Could this loop be moved into a <code>rename_fovs_in_runs</code> function? It should only need <code>processed_base_dir, cohort_path, bin_base_dir,</code> and <code>run_names</code> as arguments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>